lib/b2b/index.ts

```diff
@@ -6,7 +6,6 @@
 
 // MANUAL(exports)(TYPES)
 export type { SearchQueryOperand } from "./organizations";
-export type { B2BOrganizationsMembersOAuthProvidersMicrosoftRequest } from "./organizations_members_oauth_providers";
```

Comment:
Add this one back again -- it got removed after the first codegen, so you'll need to reset the repo or just add it back in manually once